src/main/java/core/basesyntax/strategy/impl/BalanceHandler.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.strategy.OperationHandler;
+
+public class BalanceHandler implements OperationHandler {
```

Comment:
Don't forget about the naming convention. Your realization should named `BalanceOperationHandler`
fix for all realizations